ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -391,7 +408,15 @@ pub(crate) struct Function {
     blocks: Vec<Block>,
 }
 
-impl Function {
+impl<'a> Function {
+    pub(crate) fn get_name(&self) -> &String {
```

Comment:
Pointless `get_` (I won't comment on the rest of these but you get the idea).

Reply:
[In general it's also more idiomatic to return `&str` than `&String`, but it's not a big deal.]

Reply:
Fixed in 4b6a16b0d2138270895891c980d0f5f6a1993839, then removed in c8333b87fab7833bd6584ed4234b0d27b4c545db.